pipeline-stack.ts

```diff
@@ -1,10 +1,10 @@
-import type { CdkManager } from './manager';
-import type { Account, Instance, SubInstance } from './config';
 import { Stack, StackProps, Stage, StageProps } from 'aws-cdk-lib';
-import * as pipelines from 'aws-cdk-lib/pipelines';
 import * as codebuild from 'aws-cdk-lib/aws-codebuild';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as pipelines from 'aws-cdk-lib/pipelines';
 import type { Construct } from 'constructs';
-import * as cdk from 'aws-cdk-lib';
+import type { Account, Instance, SubInstance } from './config';
+import type { CdkManager } from './manager';
```

Comment:
Your editor is trash.

Reply:
it is alphabetical by source library i think.